docs/pages/concepts/architecture/database.md

```diff
@@ -24,5 +24,5 @@ Meshery Database has several kinds of database implementations to support variou
 
 | Component      | Library                               |
 | :------------- | :------------------------------------ |
-| Bitcask        | github.com/prologic/bitcask           |
-| SQLite         | gorm.io/gorm, gorm.io/driver/sqlite   |
\ No newline at end of file
+| Bitcask        | git.mills.io/prologic/bitcask           |
```

Comment:
```suggestion
| Bitcask        | git.mills.io/prologic/bitcask         |
```

Reply:
Removing whitespace for purposes of kicking off a new build with refreshed workflow configuration (should fix some of the failed build checks).